client/src/services/resourceService.js

```
@@ -16,6 +16,13 @@ export default class ResourceService {
 		return [];
 	}
 
+	async getOne(id) {
+		const res = await this.fetch(`${ResourceService.ENDPOINT}/${id}`);
+		if (res.ok) {
+			return this._revive(await res.json());
+		}
```

Currently, the method silently fails without providing feedback or throwing an error.
Suggestion: Add an else block to handle non-OK responses, like throw an error or return a meaningful message.